packages/block-editor/src/components/block-preview/auto.js

```diff
@@ -110,7 +111,12 @@ function ScaledBlockPreview( {
 			>
 				<EditorStyles styles={ editorStyles } />
 				{ contentResizeListener }
-				<MemoizedBlockList renderAppender={ false } />
+				<div
+					dangerouslySetInnerHTML={ {
+						__html: html,
+					} }
+				></div>
+				{ /* <MemoizedBlockList renderAppender={ false } /> */ }
```

Comment:
Some loader is probably needed while fetching the preview.

Reply:
ideally it should not be needed at this point - the html should exist via the props this component gets.

Reply:
not necessarily saying but the preview function should be async, whenever that thing is in flight, we should probably show some kind of visual indicator.